met-web/src/components/engagement/form/EngagementForm.tsx

```diff
@@ -279,28 +290,19 @@ const EngagementForm = () => {
                             </MetPaper>
                         </Grid>
 
-                        <Grid item xs={12} md={5}>
-                            {creatingNewEngagement ? (
-                                <Button
-                                    fullWidth
-                                    variant="contained"
-                                    onClick={() => handleCreateEngagement()}
-                                    disabled={saving}
-                                >
-                                    Create Engagement Draft
-                                    {saving && <CircularProgress sx={{ marginLeft: 1 }} size={20} />}
-                                </Button>
-                            ) : (
-                                <Button
-                                    fullWidth
-                                    variant="contained"
-                                    onClick={() => handleUpdateEngagement()}
-                                    disabled={saving}
-                                >
-                                    Update Engagement Draft
-                                    {saving && <CircularProgress sx={{ marginLeft: 1 }} size={20} />}
-                                </Button>
-                            )}
+                        <Grid item xs={12}>
+                            <Button
+                                variant="contained"
+                                sx={{ marginRight: 1 }}
+                                onClick={() => handleSaveEngagement(false)}
```

Comment:
I think the design isn't to move to preview with the save button, it's to have another separate button to move to preview. Whether creating or updating I understood not to navigate away from the page since the user might need to do more changes.

Reply:
Saving an engagement does not navigate away, hence the "Save & preview button"

Reply:
My bad here, I misread navigate(`/engagement/form/${engagement.id}`);